homeassistant/components/ondilo_ico/config_flow.py

```diff
@@ -37,3 +50,30 @@ def logger(self) -> logging.Logger:
     def extra_authorize_data(self) -> dict:
         """Extra data that needs to be appended to the authorize url."""
         return {"scope": "api"}
+
+
+class OndiloOptionsFlowHandler(config_entries.OptionsFlow):
+    """Handle Ondilo options."""
+
+    def __init__(self, config_entry):
+        """Initialize Ondilo options flow."""
+        self.config_entry = config_entry
+
+    async def async_step_init(self, user_input=None):
+        """Manage the options."""
+        if user_input is not None:
+            return self.async_create_entry(title="", data=user_input)
+
+        return self.async_show_form(
+            step_id="init",
+            data_schema=vol.Schema(
+                {
+                    vol.Required(
+                        CONF_SCAN_INTERVAL,
```

Comment:
We don't allow scan interval as option. Every integration that supports config entries allows the user to turn off automatic polling and then the user can automate the update interval as needed with the entity service `homeassistant.update_entity`.